contrib/python-brotli/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "python-brotli"
```

Comment:
this should probably be in main/, and have a comment added into main/brotli to update it at the same time, etc